### ISSUE TITLE
Filter blocks (editor): missing label for <textarea> element

### ISSUE THREAD

**Issue by Aljullu** [woocommerce/woocommerce-gutenberg-products-block#1680]
![imatge](https://user-images.githubusercontent.com/3616980/73441278-04cb4b80-4353-11ea-8e93-26d1c8e11877.png)

Filter blocks allow editing the title via a `<textarea>`, but it's missing a label.
